src/db/gc/mod.rs

```diff
@@ -328,7 +330,7 @@ mod test {
 
         // test insufficient epochs
         gc.gc_workflow(ZERO_DURATION).await.unwrap();
-        assert!(gc.marked.is_empty());
+        assert_eq!(gc.marked.len(), 0);
```

Comment:
We don't need this change anymore, right?
```suggestion
        assert!(gc.marked.is_empty());
```